.github/workflows/terraform.yml

```diff
@@ -39,6 +39,11 @@ on:
         required: false
         default: ubuntu-latest
 
+      var_file:
+        description: Variable file with regional-specific values
```

Comment:
Wouldn't it be better to specify the description as something a bit more neutral? I mean, since you're referencing a .tfvars file, it could be anything. Not just region specific stuff.

Reply:
I agree 👍 

How about something like this?

> The path, relative to the working directory, of a variable definitions (`.tfvars`) file containing values for variables declared in the root module.

Reply:
^ This is very similar to the description we already use for input `backend_config`:

> The path, relative to the working directory, of a configuration file containing the remaining arguments for a partial backend configuration.

Reply:
> I agree 👍
> 
> How about something like this?
> 
> ```
> The path, relative to the working directory, of a variable definitions (`.tfvars`) file containing values for variables declared in the root module.
> ```

If you agree, I'd like to move "(`.tfvars`)" after "file" so it doesn't split the sentence

Reply:
My intention was that you could read it as either "variable definitions file" or "`.tfvars` file".

Also, I took it word-for-word from [this heading](https://developer.hashicorp.com/terraform/language/values/variables#variable-definitions-tfvars-files) in the official docs.